chain/evm/client.go

```diff
@@ -112,22 +117,26 @@ type Client struct {
 	addr     ethcommon.Address
 	keystore *keystore.KeyStore
 
-	conn   ethClientConn
+	conn   EthClientConn
```

Comment:
I didn't make the variable public, just the interface.  I added a getter to get the conn as well.  In order to make calls on the interface from a different package, it needs to be public.

I'm using this to build query filters, rather than using the more restrictive filtering built in elsewhere.

Reply:
But it's not used outside the package, is it? I cannot find any references outside the `evm` package?

Reply:
You're totally right.  I've switched it back to private now